io.github.unknownskl.greenlight.yml

```diff
@@ -0,0 +1,79 @@
+app-id: io.github.unknownskl.greenlight
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+platform-extensions:
+  - org.freedesktop.Platform.VAAPI.Intel
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --allow=bluetooth
+  - --device=all
+  - --filesystem=/run/udev:ro
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: greenlight
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/greenlight/flatpak-node/cache
+        npm_config_cache: /run/build/greenlight/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+        yarn_config_cache: /run/build/greenlight/flatpak-node/yarn-mirror
+    build-commands:
+      - cp -a ../extralib -t /app/
+      - yarn --offline
+      - yarn run flatpak-build --offline -- $ELECTRON_BUILDER_ARCH_ARGS --linux
```

Comment:
where is `ELECTRON_BUILDER_ARCH_ARGS` defined?

Reply:
It is a leftover from an example flatpak file. Will remove it, thanks.